client/state/utils/README.md

```diff
@@ -0,0 +1,18 @@
+Utils
+========
+
+A module for those little helpers that need to interact woth the store.
```

Comment:
As you probably noticed reviewing my sloppy PRs, attention to detail is not my forte :/